run/do-harvest

```diff
@@ -23,9 +23,20 @@ $HOME/run/pop2illiad $illiad_date
 
 cat $LOG/harvest.log | mailx -s 'Harvest Log' sul-unicorn-devs@lists.stanford.edu
 
-mv $LOG/harvest.log $LOG/harvest.log.$DATE
+if [ -e $LOG/harvest.log.$DATE ]
+  then
+  mv $LOG/harvest.log $LOG/harvest.log.`date +%Y%m%d%H%M`
```

Comment:
Overkill?  I think you could just replace 'mv $LOG/harvest.log $LOG/harvest.log.$DATE'
with
mv $LOG/harvest.log $LOG/harvest.log.`date +%Y%m%d%H%M`

all the time, not just when the harvest.log.$DATE file already exists?  Basically have a datetime stamped file for each harvest run we do.

Would need to also datetime stamp the harvest.out file too, and now illiad files